core/pkg/service/flag-evaluation/flag_evaluator.go

```diff
@@ -45,6 +45,7 @@ func NewFlagEvaluationService(log *logger.Logger,
 	}
 }
 
+// nolint:dupl
```

Comment:
Nit: I recommend moving this to the top of the file and adding another comment here saying eventually the whole thing can be removed.